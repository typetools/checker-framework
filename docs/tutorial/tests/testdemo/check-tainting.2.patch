Index: src/net/eyde/personalblog/struts/action/ReadAction.java
===================================================================
--- src/net/eyde/personalblog/struts/action/ReadAction.java	(revision 3589)
+++ src/net/eyde/personalblog/struts/action/ReadAction.java	(working copy)
@@ -45,7 +45,7 @@
         String forward = "readposts";
 
         // Get request parameters
-        String reqCategory = cleanNull(request.getParameter("cat"));
+        String reqCategory = validate(cleanNull(request.getParameter("cat")));
 
         // Get instance of PersonalBlog Service
         PersonalBlogService pblog = PersonalBlogService.getInstance();
