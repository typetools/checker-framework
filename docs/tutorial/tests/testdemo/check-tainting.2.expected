
clean:
Deleting directory /home/mernst/research/types/checker-framework-REAL/tutorial/eclipse-projects/personalblog-demo/bin

check-tainting:
Created dir: /home/mernst/research/types/checker-framework-REAL/tutorial/eclipse-projects/personalblog-demo/bin
Compiling 2 source files to /home/mernst/research/types/checker-framework-REAL/tutorial/eclipse-projects/personalblog-demo/bin
javac 1.8.0-jsr308-1.9.9

BUILD SUCCESSFUL
Total time: 2 seconds
